storybook/stories/Examples/cartesian/Line/LineWithPoints.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Line, ResponsiveContainer, ComposedChart } from '../../../../../src';
+import { coordinateData } from '../../../data';
+
+export default {
+  title: 'Examples/cartesian/Line/With Points',
+};
+
+export const WithPoints = {
+  render: () => {
+    const [surfaceWidth, surfaceHeight] = [600, 300];
+    return (
+      <ResponsiveContainer width="100%" height={surfaceHeight}>
```

Comment:
something not quite right with this story but not sure what it is, either way nothing is displaying in the storybook